vmm/src/lib.rs

```diff
@@ -2031,28 +2171,40 @@ impl RequestHandler for Vmm {
 
                     if !started {
                         warn!("Migration not started yet");
-                        Response::error().write_to(&mut socket)?;
+                        Response::error().write_to(&mut socket).map_err(|e| {
+                            MigratableError::MigrateReceive(anyhow!(
+                                "Error sending response: {}",
+                                e
+                            ))
+                        })?;
```

Comment:
> I don't see why this change is needed.

This is to ensure that any errors that occur while writing to the socket are properly caught and handled, instead of causing a panic.

Reply:
The `?` should propagate the error - was something at a higher level not handling this correctly?